core/frontend/vitest.config.mts

```diff
@@ -0,0 +1,68 @@
+import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+export default defineConfig({
+  test: {
+    dir: "src",
+    setupFiles: "./src/test/setupTests.ts",
+    // include: ["**/<testfileName>.test.ts"], // to adhere to chai's describe.only/it.only
+    browser: {
+      provider: "playwright",
+      enabled: true,
+      name: "chromium",
+      headless: true,
+      screenshotFailures: false
```

Comment:
this might be useful to enable

Reply:
it's enabled by default, I turned it off while working until I can figure out an apt subfolder to save these screenshots (and update gitignore accordingly...)

Reply:
It'll be done in a separate PR. Issue opened: https://github.com/iTwin/itwinjs-core/issues/7277